src/Domain/Model/Product/ProductTaxa/ProductTaxon.php

```
@@ -0,0 +1,66 @@
+
+    public function getMappedData(): array
+    {
+        return [
+            'product_id' => $this->productId->get(),
+            'taxon_id' => $this->taxonId->get(),
+            'state' => $this->state->value,
+            'data' => json_encode($this->data),
+        ];
+    }
+
+    public static function fromMappedData(array $state, array $aggregateState, array $childEntities = []): static
```

PHPMD (2.15.0)

Avoid unused parameters such as '$childEntities'. (Unused Code Rules)

(UnusedFormalParameter)